frontend/app/src/components/history/events/HistoryEventsIdentifier.vue

```diff
@@ -21,18 +21,24 @@ const { is2xlAndUp } = useBreakpoint();
 const translationKey = computed<string>(() => {
   // consider an evm swap event as a case of evm event
   // as they are both evm events and have the same header
-  let entryType = get(event).entryType;
+  const eventVal = get(event);
+  let entryType = eventVal.entryType;
   if (entryType === HistoryEventEntryType.EVM_SWAP_EVENT)
     entryType = HistoryEventEntryType.EVM_EVENT;
+
+  if (entryType === HistoryEventEntryType.HISTORY_EVENT && 'txHash' in eventVal) {
+    entryType = HistoryEventEntryType.EVM_EVENT;
+  }
```

Comment:
Do we have cases of normal history events with transaction hashes?

Reply:
normal history events don't have hashes, only bitcoin txs

Reply:
Can you make a note for that it would make sense yes